src/test/java/eu/europeana/metis/sandbox/service/workflow/HarvestServiceImplTest.java

```diff
@@ -6,10 +6,7 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
just leave 
```
import static org.mockito.Mockito.mock;
import static org.mockito.Mockito.spy;
import static org.mockito.Mockito.times;
import static org.mockito.Mockito.verify;
import static org.mockito.Mockito.when;
```
and remove 
```
import static org.mockito.Mockito.*
```
this is why is important: https://rules.sonarsource.com/java/RSPEC-2208

why should we avoid it?

Explicit imports clearly states what external classes the current class is directly using, provided that you don’t leave redundant imports in your code.
If there are two classes with the same name from different packages, it can introduce collision when using wildcard imports.
When multiple people are working in a project, wildcard imports can create confusion as to which classes are actually imported.

Reply:
Thanks for the explanation! I'll update the imports